### ISSUE TITLE
is ouibounce not working for mobile and tablet views? is it because there is no mouse leave event? 

### ISSUE THREAD

**Issue by SoubhagyaB** [carlsednaoui/ouibounce#123]
Please let me know if any body have any solution to show the ouibounce modal window in mobile and tablet views. Any help will be appreciated. Thanks in advance

**Comment by pjatx** [comment]
Yeah - it doesn't support touch devices. One approach can be found here: https://github.com/carlsednaoui/ouibounce/pull/96 which seems to work when folks scroll very quickly up the page. 

Other options are detecting mobile devices and firing at a certain time, otherwise use normal desktop behavior.
